docs/get-started/yarn-gpu.md

```diff
@@ -117,3 +117,39 @@ On all masters:
 ```bash
 sudo systemctl restart hadoop-yarn-resourcemanager.service
 ```
+
+Note: If `cgroup` is mounted on `tmpfs` and a node is rebooted,
+the cgroup directory permission gets reverted.  In this case, 
+cgroup permission can be updated as follows:
+```bash
+chmod a+rwx -R /sys/fs/cgroup/cpu,cpuacct
+chmod a+rwx -R /sys/fs/cgroup/devices
+```
+or the operation can be added in the systemd scripts:
+Create mountCgroup scripts:
+```bash
+sudo bash -c "cat >/etc/systemd/system/mountCgroup.service" <<EOF
+[Unit]
+Description=startup
+[Service]
+ExecStart=/etc/mountCgroup.sh
+Type=oneshot
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo bash -c "cat >/etc/mountCgroup.sh" <<EOF
+#!/bin/sh
+chmod a+rwx -R /sys/fs/cgroup/cpu,cpuacct
+chmod a+rwx -R /sys/fs/cgroup/devices
+EOF
+
+sudo chmod 644 /etc/systemd/system/mountCgroup.service
+sudo chmod 777 /etc/mountCgroup.sh
```

Comment:
This should not be 777. Why isn't it 644 like `mountCgroup.service`?

Reply:
644 will get an error
`systemd[2523]: startup.service: Failed at step EXEC spawning /etc/startup.sh: Permission denied`
but 655 works, update to 655